content/en-us/reference/engine/classes/UnreliableRemoteEvent.yaml

```diff
@@ -65,31 +164,64 @@ methods:
     thread_safety: Unsafe
 events:
   - name: UnreliableRemoteEvent.OnClientEvent
-    summary: ''
-    description: ''
+    summary: |
+      Fires from a `Class.LocalScript` when either
+      `Class.UnreliableRemoteEvent:FireClient()|FireClient()` or
+      `Class.UnreliableRemoteEvent:FireAllClients()|FireAllClients()` is called on the
+      same `Class.UnreliableRemoteEvent` instance from a `Class.Script`, although
+      this firing is not guaranteed even if one of the above methods are called. This
+      can occur due to packet loss or to maintain optimal engine performance.
+    description: |
+      Fires from a `Class.LocalScript` when either
+      `Class.UnreliableRemoteEvent:FireClient()|FireClient()` or
+      `Class.UnreliableRemoteEvent:FireAllClients()|FireAllClients()` is called on the
+      same `Class.UnreliableRemoteEvent` instance from a `Class.Script`, although
+      this firing is not guaranteed even if one of the above methods are called. This
+      can occur due to packet loss or to maintain optimal engine performance.
+
+      It is not guaranteed that the order of events will match the order of FireServer() calls.
```

Comment:
I think you mean FireClient() and FireAllClients() here right? Not FireServer()?